src/main/java/krona/task/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package krona.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a task with a deadline that the user
+ * can input date and time.
+ */
+public class Deadline extends Task {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
```

Comment:
Maybe you can change the formatter name to be in all caps instead, since it has the final modifier, it means its a constant, so we have to put its name in all caps according to the coding standard